taps-api/data/venues.js

```diff
@@ -28,19 +28,22 @@ module.exports = {
     if (typeof lng !== 'string' || lng.trim() === '') throw new Error('lng must be a non-empty string');
 
     const venuesCollection = await venues();
+    // schmema changed to be in geojson format
     const newVenue = {
-      _id: uuid.v4(),
-      name,
-      city,
-      state,
-      country,
-      lat,
-      lng,
-      beersAvailable: []
+      type: "Point",
+      coordinates: [lng, lat],
```

Comment:
One thing that needs to be changed here is that lng and lat are both strings atm. We need to use a parse function to make these numbers.